frontend/src/libs/components/header/styles.module.scss

```diff
@@ -0,0 +1,27 @@
+@import url("../../../assets/css/variables.scss");
```

Comment:
No need to include this file. It is imported in global styles.scss file, which is already imported to the app.